docs/api-testing/integrations/yaml.md

```diff
@@ -77,6 +77,10 @@ sauce:
 
 Specifies through which Sauce Labs data center tests will run. Valid values are: `us-west-1` or `eu-central-1`.
 
+:::note
+`region` is mandatory and should be set either in the Sauce Config file or by using the `--region` flag.
```

Comment:
It's just confusing to say `region` is mandatory when above its marked as OPTIONAL. Maybe something like this?
```suggestion
If you do not specify a region in your config file, you must set it when running your command with the `--region` flag.
```